src/Auth.js

```
@@ -69,6 +69,24 @@ export default class Auth {
       });
   }
 
+  getCurrentUser(callback) {
+    axios({
+      method: 'GET',
+      headers: {
+        Authorization: 'Bearer '.concat(localStorage.getItem('access_token'))
+      },
+      url: `${process.env.REACT_APP_API_URL}/user`,
+      params: { auth0_id: localStorage.getItem('user_id') }
+    })
+      .then(function(response) {
+        let user = response.data[0];
+        callback(null, user);
+      })
+      .catch(function(error) {
+        console.error(error);
```

Should bubble this error back up: `callback(error)`